addons/mail/static/src/models/notification_group_view.js

```diff
@@ -20,9 +20,6 @@ registerModel({
                 return;
             }
             this.notificationGroup.openDocuments();
-            if (!this.messaging.device.isSmall) {
```

Comment:
I think the condition is still necessary if `openDocuments` is opening a thread instead, as the auto-close seems to be on the "else" case only?

Reply:
Indeed, I was a bit too quick and missed the `(!this` and `(this`.

I changed the fix by keeping same logic but just storing `messaging` before `openDocuments`, so that there's no accessor on `this` from destroyed record.